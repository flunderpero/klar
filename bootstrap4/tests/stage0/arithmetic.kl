--- expected

# compiler output:
# exit code:
0
# stderr:
# stdout:
5
3

/expected ---

fn main():
    print(1 + 2 * 3 - 4 / 2)
    print( ( (1 + 2) * 3 - 4 + 1) / 2)
end
