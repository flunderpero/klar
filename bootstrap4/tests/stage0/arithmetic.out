5
---
exit code: 0
